server/controllers/questions.js

```diff
@@ -17,8 +17,8 @@ exports.getQuestions = (req, res) =>{
 };
 
 exports.getAnswers = (req, res) =>{
-
-  axios.get('https://app-hrsei-api.herokuapp.com/api/fec2/hr-rpp/qa/questions/640996/answers', {
+  let qid=req.params.qid;
+  axios.get(`https://app-hrsei-api.herokuapp.com/api/fec2/hr-rpp/qa/questions/${qid}/answers`, {
```

Comment:
can also make a variable such as const = AtelierAPI for your API link to keep the code DRY, then just use the variable and concatenate with your specific API parameter